src/screens/cars/Cars.tsx

```diff
@@ -162,6 +168,13 @@ export default class Cars extends SelectableList<Car> {
         <View style={transactionStyles.searchBar}>
           <SimpleSearchComponent onChange={async (searchText) => this.search(searchText)} navigation={navigation} />
         </View>
+        {!isModal && (
```

Comment:
The Add Car button is taking too much space - move it to the header?
<img width="429" alt="2021-10-04_10-34-28" src="https://user-images.githubusercontent.com/25266945/135819637-fb3bcc46-071d-405e-ae3c-214333226e21.png">

Reply:
I was thinking about a floating bottom-right icon ?

Reply:
I had thought the same thing lol, but it maybe hide part of the list at the bottom.
Give it a try :)

Reply:
it looks very nice I think
The squared one is an alternative (I have exactly the same in my phone contacts list, before it was a circle

Circle:
<img width="439" alt="Screenshot 2021-10-04 at 13 39 41" src="https://user-images.githubusercontent.com/32574035/135846542-2a1fb85a-1bbf-4d64-aefc-89f5a7f15abe.png">
)
![1633347782870 copy](https://user-images.githubusercontent.com/32574035/135846578-53eef1f9-89bc-4182-b573-5e2260f4213e.png)


Square:

![Screenshot_2021-10-04-13-47-27-699_com emobility copy](https://user-images.githubusercontent.com/32574035/135846601-75b26338-7038-4fbf-bf50-0b8c4247604d.png)

Reply:
Circle is nice!